### PR TITLE
Tweak mapgen for animal pound

### LINKED ISSUE #38000: Animal Pound building has bad placement feature.
the animal pound has a vehicle, however the vehicle is placed at the back and hence not necesarrily connected to an open street.

Picture 

# Screenshots

2020-02-13 20_50_24-Cataclysm_ Dark Days Ahead - 0 D-12062-ge7271b0

### PR DESCRIPTION
**Pull request by arijust**
#### Summary

```SUMMARY: Bugfixes "Tweak mapgen for animal pound"```
#### Purpose of change

Close #38000
#### Describe the solution

Tweak existing map.
#### Describe alternatives you've considered

Make entirely new one and close this PR.

#### Additional context

It should be all.

### CODE REVIEW

**Comment by curstwist** [comment]
can you add some screenshots of your changes?

**Comment by I-am-Erk** [comment]
Can post a picture or two of your work?

**Comment by arijust** [comment]
Unfortunately, I am unable to post any images here as my router is broken piece of garbage.
Sorry.

**Comment by cosmiccoincidence** [comment]
Seems they simply removed the part of the building obstructing the vehicle to the road. I'm referring to the outdoor dog cages.

I think an alternative to this is simply swapping the locations of the vehicle and outdoor cages as opposed to outright removing them as they are kinda neat. But this aesthetic could also be preserved by making another animal shelter map so we have 2 variants.

**Comment by I-am-Erk** [comment]
Yes, that's what it was looking like to me, and I'd argue the animal cages are a far more important part of the pound than the vehicle spawn.

**Comment by curstwist** [comment]
hey, i went ahead and loaded this for screenshots.
animal pound 1


The vehicle spawn location (assuming its still in the back) isn't alleviated:
problem

You'll need to spawn it in the front of the building and push the building back to the edge of its map, or redo a size and put it there.

**Comment by I-am-Erk** [comment]
If doing that, you should also move the building closer to the edge of the map and put those animal cages off to one side, because as I said, I feel pretty strongly that they're more important than the vehicle. There's room for them... but like I said in the issue, it means redrawing most of the map.

File: data/json/mapgen/animalpound.json [hunk 1]
```diff
@@ -27,30 +27,29 @@
...
+        "'-|=====|l-ll+llTllTll-'",
+        "'-|lllllal-ll-ll#ll#ll-'",
+        "'-|=====|l+ll----------'",
+        "'-|lllllal-ll-FFss#kkk-'",
+        "'-|=====|l-ll-Qlllllll-'",
+        "'-|lllllal-ll+llTllTll-'",
+        "*-|=====|R-ll-ll#ll#ll-'",
+        "'----------M-----------4",
+        "'~~~~~~~~~~~~~~~~~~~~ff~",
+        "''''''''''''''''''''''''"
```
Comment (BevapDin) [inline data/json/mapgen/animalpound.json]: There is a row missing. The original map had 24 rows, the new one only has 23 rows.

**Comment by I-am-Erk** [comment]
Did the problem of the deleted cages get resolved? I wasn't quite ready to merge this yet

**Comment by cosmiccoincidence** [comment]
> Made second vet office.

I don't see a second vet office added...

**Comment by arijust** [comment]
Darn it. It wasn't ready to merge yet.

**Comment by cosmiccoincidence** [comment]
You can put "[WIP]" at the beginning of the PR's title to let mergers know not to merge until you remove "[WIP]" from the title.

**Comment by arijust** [comment]
I guess, I will need to add missing parts in other PR.

**Pull request merged**